index.bs

```diff
@@ -271,6 +272,14 @@ urlPrefix: https://fetch.spec.whatwg.org; spec: FETCH;
       <p>The impact of these states on the processing model of various [=fetch|fetches=] is defined
       in more detail throughout this specification, in [[FETCH|Fetch]], and in
       [[PRIORITY-HINTS|Priority Hints]].</p>
+      <p>The <dfn export>computed fetch priority</dfn> of a supported element is defined as:</p>
+      <ol>
+        <li><p>If the element is [=render-blocking=] and the current state of its
+          [=fetchpriority attribute=] is [=low=], then the [=computed fetch priority=] is the
+          [=auto=] state;</p>
```

Comment:
Or should I bump both `auto` and `low` to `high`? Because it seems that Chrome treats some `auto` as `low`:

https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/http/tests/priority-hints/link-preload-dynamic-insertion.html;bpv=0